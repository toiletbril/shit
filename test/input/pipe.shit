echo 1 | cat | cat | cat;
echo 2 | cat && echo 3 | cat;
echo 4 | false || echo 4;
