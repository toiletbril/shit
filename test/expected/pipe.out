[Sequence]
  [SequenceNode Simple]
    [ExecPipeSequence]
      [Exec "echo 1"]
      [Exec "cat"]
      [Exec "cat"]
      [Exec "cat"]
  [SequenceNode Simple]
    [ExecPipeSequence]
      [Exec "echo 2"]
      [Exec "cat"]
  [SequenceNode And]
    [ExecPipeSequence]
      [Exec "echo 3"]
      [Exec "cat"]
  [SequenceNode Simple]
    [ExecPipeSequence]
      [Exec "echo 4"]
      [Exec "false"]
  [SequenceNode Or]
    [Exec "echo 4"]
1
2
3
4
[Code 0]
